honeybee_energy_standards/lib/constructions.py

```diff
@@ -0,0 +1,63 @@
+"""Extend the honeybee_energy construction library."""
+from honeybee_energy.construction.opaque import OpaqueConstruction
+from honeybee_energy.construction.window import WindowConstruction
+from honeybee_energy.lib.constructions import _idf_opaque_constructions, _idf_window_constructions
+
+import os
+import json
+
+
+# load the standards gem data of construction to Python dictionaries.
+_data_dir = os.path.join(os.path.dirname(__file__), '../data')
+
+_opaque_dir = os.path.join(_data_dir, 'opaque_construction.json')
+with open(_opaque_dir, 'r') as f:
+    _opaque_constr_standards_dict = json.load(f)
+
+_window_dir = os.path.join(_data_dir, 'window_construction.json')
+with open(_window_dir, 'r') as f:
+    _window_constr_standards_dict = json.load(f)
```

Comment:
I would suggest to find a solution to cache these values. With the current structure they will be loaded on each import. This can slow down the overall process. Since this will be a design change I suggest to open an issue for this case and address it in the future.

Reply:
Is that right? I thought modules are automatically cached when they are imported. Let me do a quick test and see.

Reply:
What you said here does not appear to be correct. I put a piece of test code in:
![image](https://user-images.githubusercontent.com/5567574/67537645-4ee4c700-f6aa-11e9-91d6-0eee795b0a2c.png)

I try importing a module twice:
![image](https://user-images.githubusercontent.com/5567574/67537674-6623b480-f6aa-11e9-8a35-fa9e2a8f49f7.png)

You see I get only one 'oh no!' printed.
And that's a good thing because I feel like nearly all of the code would be so much less efficient if it re-compiled and re-ran everything each time it was imported.

Reply:
That is correct in standard Python you are right but the modules won't be cached in all platforms (e.g. Dynamo).

Reply:
I just ran another test from Grasshopper with two separate components and it looks like Grasshopper is using "standard Python" as you are defining it here (the data is only loaded once). That's a really poor design if Dynamo re-compiles and re-runs the imported module each and every time you import it.  If that's true, then there's no way that our extension libraries are going to translate to Dynamo since, if it re-compiles and re-runs everything each time someone imports something, then each component is going to take at least 5 seconds to run through all of the loading of extensions before it runs the real code.

In any case, I'd say that, much in the same way that we aren't using properties like `isAnalysisPeriod` in our objects right now, we shouldn't let the poor design of Dynamo lead to poor design within our core libraries here.